engine/rest-client/src/main/java/org/datacleaner/restclient/ComponentController.java

```diff
@@ -47,12 +47,8 @@
 
     /**
      * It creates a new component with the provided configuration, runs it and returns the result.
-     * @param tenant
-     * @param name
-     * @param processStatelessInput
-     * @return
      */
-    public ProcessStatelessOutput processStateless(final String tenant, final String name,
+    public ProcessStatelessOutput processStateless(final String tenant, final String name, String outputFormat,
```

Comment:
Maybe would be better with an enum here so we control which outputFormats are applicable.